fineract-savings/src/main/java/org/apache/fineract/interoperation/data/InteropIdentifierData.java

```diff
@@ -19,41 +19,28 @@
 package org.apache.fineract.interoperation.data;
 
 import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
 import org.apache.fineract.interoperation.domain.InteropIdentifier;
 import org.apache.fineract.interoperation.domain.InteropIdentifierType;
 
+@Getter
+@AllArgsConstructor
 public class InteropIdentifierData {
 
     @NotNull
-    private final InteropIdentifierType idType;
```

Comment:
Why do you remove the final tag?

Reply:
@adamsaghy I think the idea is to better use the builder pattern instead of a gigantic (depending on number of class attributes) constructor with "57" string parameters (I am exaggerating).

Reply:
Builder pattern might be okay, but it is not used here. so its just a simple dto which is created once and from that moment it is immutable.